Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7-slim AS base
+
+FROM base AS python-deps
+RUN pip install pipenv
+RUN apt-get update && apt-get install -y --no-install-recommends gcc
+COPY Pipfile .
+COPY Pipfile.lock .
+RUN PIPENV_VENV_IN_PROJECT=1 pipenv install --deploy
+
+FROM base AS runtime
+COPY --from=python-deps /.venv /.venv
+ENV PATH="/.venv/bin:$PATH"
+COPY . .
+
+EXPOSE 5000
+
+ENTRYPOINT ["python3", "app.py"]
```

Comment:
nitpick - let's add a new line here.

Reply:
Just for knowledge , do we have a new line at the end of file because some interpreters consider the line last new line as the EOF?

Reply:
https://stackoverflow.com/a/729795/8184394